### PR TITLE
Add XRD allocation to genesis txn

### PR DESCRIPTION
**Pull request by LukasGasior1**

### CODE REVIEW

File: core-rust-bridge/src/main/java/com/radixdlt/rev2/ComponentAddress.java
Comment (dhedey) [inline core-rust-bridge/src/main/java/com/radixdlt/rev2/ComponentAddress.java:133]: I'd slightly rather this was done through a static call into rust, as I believe this mapping is going to change in future

File: core-rust/core-api-server/Cargo.toml
Comment (dhedey) [inline core-rust/core-api-server/Cargo.toml:26]: Now that https://github.com/radixdlt/radixdlt-scrypto/pull/790/files has merged, can we create a new tag `betanet-v2-xyz` n the scrypto repo and use that before merging :)

**Review by dhedey — APPROVED** [review]
Nice, and thanks for updating the test.

Happy to merge once markups are done - also, would you be able to add the override of this property to the staking address in this PR, as discussed on slack?

File: core-rust-bridge/src/main/java/com/radixdlt/rev2/ComponentAddress.java
Comment (LukasGasior1) [reply core-rust-bridge/src/main/java/com/radixdlt/rev2/ComponentAddress.java:133]: good point...and in fact it was already there :D `Address.virtualAccountAddress`

**Pull request merged**
